queries/wgsl/indents.scm

```diff
@@ -1,10 +1,17 @@
 [
-  (parameter_list)
   (compound_statement)
   (loop_statement)
   (struct_declaration)
 ] @indent.begin
 
+((parameter_list) @indent.begin
+  (#set! indent.immediate)
+  (#set! indent.start_at_same_line))
+
+(function_declaration
+  "(" @indent.begin
+  (#set! indent.immediate))
```

Comment:
Does it cover for things like
```
fn foo
(
   a,
   b,
   c
)
```

You may or may not cover for this, it's up to you. I feel like this is too much of an edge case to be caring about

Reply:
It will indent all the params properly in this case (no function body) but I think the `"("` indentation is not applied (so while editing users have to indent themselves). Maybe could be looked into in the future

Reply:
Actually I have just added support for this